range.py

```diff
@@ -0,0 +1,7 @@
+
+def check_range(value_list):
+  count = 1
+  for i in range(len(value_list)-1):
+    if value_list[i] == value_list[i+1]-1:
```

Comment:
value with diff of 1 is considered as consecutive, but same value is not handled.
Please add test case with same values too.